frontend/src/scenes/early-access-features/EarlyAccessFeature.tsx

```diff
@@ -303,37 +305,57 @@ function PersonList({ earlyAccessFeature }: PersonListProps): JSX.Element {
         ],
     }
     const logic = personsLogic(personsLogicProps)
-    const { persons } = useValues(logic)
+    const { persons, personsLoading, listFilters } = useValues(logic)
+    const { loadPersons, setListFilters } = useActions(logic)
     const { featureFlag } = useValues(featureFlagLogic({ id: earlyAccessFeature.feature_flag.id || 'link' }))
 
     return (
         <BindLogic logic={personsLogic} props={personsLogicProps}>
             <h3 className="text-xl font-semibold">Opted-In Users</h3>
-            <PersonsScene
```

Comment:
@Twixes - it's now annoying that we have to replicate all the logic from the scene (to add the filter & search on top). Is there another component we could use? Or does it make sense to make Search + Filter + List a component that's used inside the scene & here too?

Reply:
👆 agreed, originally why I used the scene

Reply:
Can we get this in in the meantime?

Reply:
yes!